common/changes/office-ui-fabric-react/laxmika-DropHintIconFix_2018-08-07-16-02.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Changing the drophint ICON",
+      "type": "patch"
```

Comment:
This changes the global classnames that someone might be relying on... is that necessary?

Reply:
If so, it's should be a minor, not a patch (actually changing contracts - remove + add technically would be a breaking change...)

Reply:
This CSS classname is specific to drophint of column drag and drop and now that is changed from circle to caret, so i don't think it would break any other CSS.  If you still think, its better to keep the same classname, i can revert classname changes.

Reply:
@kenotron  Kept both the classes now